thoth/analyzer/cli.py

```diff
@@ -78,6 +79,7 @@ def print_command_result(click_ctx: click.core.Command,
         'analyzer_version': analyzer_version,
         'distribution': distro.info(),
         'arguments': _get_click_arguments(click_ctx),
+        'duration': int(duration),
```

Comment:
```python
        'duration': int(duration) if duration is not None else None,
```

Otherwise:

```python
>>> int(None)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: int() argument must be a string, a bytes-like object or a number, not 'NoneType'
```

Reply:
Thanks, good catch, :100: !